Task_HelloWorld/index.html

```diff
@@ -1,11 +1,14 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Hello World Problem</title>
-    <link rel="stylesheet" href="index.css" />
-  </head>
```

Comment:
Line 6: The closing center tag </center> should come before the closing h1 tag </h1>.